src/services/minecraft.service.ts

```diff
@@ -111,19 +116,31 @@ export class MinecraftService {
   }
 
   private async getScript(flavor: string, version: string): Promise<string> {
-    // GET HEAD, FLAVOR, AND TAIL PATHS
-    const headPath = resolve(__dirname, `../scripts/head.sh`);
-    const tailPath = resolve(__dirname, `../scripts/tail.sh`);
-    const flavorPath = resolve(__dirname, `../scripts/${flavor}.sh`);
-    // READ SCRIPT FILES
-    let headScript = await asyncReadFile(headPath, 'utf8');
-    const tailScript = await asyncReadFile(tailPath, 'utf8');
-    let flavorScript = await asyncReadFile(flavorPath, 'utf8');
+    // GET SCRIPT PATH
+    const scriptPath = resolve(__dirname, `../scripts/setup.sh`);
+
+    // READ SCRIPT FILE
+    let script = await asyncReadFile(scriptPath, 'utf8');
+
+    // FETCH URL BASED ON FLAVOR
+    let flavor: MinecraftFlavor;
```

Comment:
```suggestion
    let minecraftFlavor: MinecraftFlavor;
```

Oops didn't realize we already had flavor in this context